### DISCUSSION TITLE
Prometheus Integration

### DISCUSSION THREAD

**Discussion by Nexxed** [alexjustesen/speedtest-tracker#24] [Ideas]
This is an open issue on the repo of the previous project and it would be cool to both support InfluxDB and Prometheus (for those that use one or the other):
https://github.com/henrywhitaker3/Speedtest-Tracker/issues/658

**Comment by alexjustesen** [comment]
I like, I'll get it on the roadmap

**Reply by jeffbyrnes** [reply]
Awesome! This would also allow pushing these metrics into a service like Datadog, which has an OpenTelemetry endpoint, meaning it’s compatible with Prometheus.
